liota/identity/qos_conf.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
1) Rename QoSConf to QoSDetails
2) This file cannot be in the identity layer. Move the class to mqtt.py